drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -620,6 +674,224 @@ def self.sections(report) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticC
+        },
+        income_source_amounts_match_total_at_entry: {
+          title: 'Total Monthly Income Matches Sources at Entry',
+          description: 'Total monthly income matches the sum of the sourced income at entry',
+          required_for: 'Adults',
+          detail_columns: default_detail_columns + [
+            :income_from_any_source_at_entry,
+            :total_monthly_income_at_entry,
+            :total_monthly_income_from_source_at_entry,
+          ],
+          denominator: ->(item) { adult?(item) },
+          limiter: ->(item) {
+            return false unless adult?(item)
+
+            total_monthly_income = item.total_monthly_income_at_entry || 0
+            source_income = item.total_monthly_income_from_source_at_entry || 0
+            total_monthly_income != source_income
+          },
+        },
+        total_monthly_income_at_exit: {
+          title: 'Total Monthly Income at Exit',
+          description: 'Total Monthly Income at exit matches Income from Any Source at exit',
+          required_for: 'Adults',
+          detail_columns: default_detail_columns + [
+            :income_from_any_source_at_exit,
+            :total_monthly_income_at_exit,
+          ],
+          denominator: ->(item) { item.income_at_exit_expected == true },
+          limiter: ->(item) {
+            return false unless item.income_at_exit_expected == true
+
+            has_any_source_of_income_at_exit = item.income_from_any_source_at_exit.present? && item.income_from_any_source_at_exit == 1
+            has_monthly_income_at_exit = item.total_monthly_income_at_exit.present? && item.total_monthly_income_at_exit > 0
+
+            # any source at exit XOR reported monthly income at exit
+            has_any_source_of_income_at_exit ^ has_monthly_income_at_exit
+          },
+        },
+        income_source_amounts_match_total_at_exit: {
+          title: 'Total Monthly Income Matches Sources at Exit',
+          description: 'Total monthly income matches the sum of the sourced income at exit',
+          required_for: 'Adults',
+          detail_columns: default_detail_columns + [
+            :income_from_any_source_at_exit,
+            :total_monthly_income_at_exit,
+            :total_monthly_income_from_source_at_exit,
+          ],
+          denominator: ->(item) { adult?(item) },
+          limiter: ->(item) {
+            return false unless adult?(item)
+
+            total_monthly_income = item.total_monthly_income_at_exit || 0
+            source_income = item.total_monthly_income_from_source_at_exit || 0
+            total_monthly_income != source_income
+          },
+        },
+        no_veteran_in_household: {
+          title: 'No Veteran in Household',
+          description: 'No member of the household is a veteran',
+          required_for: 'Household',
+          detail_columns: default_detail_columns + [
+            :hh_veteran_count,
+          ],
+          denominator: ->(_item) { true },
```

Comment:
I _think_ this is only valid for some funding sources.  So the denominator will need to account for that.  Not every enrollment ever needs a veteran.